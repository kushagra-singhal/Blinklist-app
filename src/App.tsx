import React from 'react';
import { CssBaseline, ThemeProvider } from "@mui/material";
import './App.css';
import Template from './components/Template/TemplateComponent';
import theme from "./Themes/main";

// import MyLibrary from './components/Pages/MyLibrary';
// import HeaderComponent from './components/organisms/Header';



function App() {
  
  
  
  return(
    <div data-testid="App1" className="App">


    <ThemeProvider theme={theme}>
      <CssBaseline />
      <Template />
      

     
    </ThemeProvider>

    </div>
  );

}





export default App;

