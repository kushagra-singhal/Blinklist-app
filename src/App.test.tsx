import React from 'react';
import renderer from 'react-test-renderer';
// import { render, screen } from '@testing-library/react';
import App from './App';
import { CssBaseline, ThemeProvider } from '@mui/material';
import theme from './Themes/main';


test('renders the App Component', () => {
  const component = renderer.create(
    <ThemeProvider theme={theme}>
      <CssBaseline />
      <App />
    </ThemeProvider>
  );
  expect(component).toMatchSnapshot();
});
