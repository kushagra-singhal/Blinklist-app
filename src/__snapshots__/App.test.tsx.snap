// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the App Component 1`] = `
<div
  className="App"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg css-14uxf7f-MuiContainer-root"
  >
    <span
      className="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-1gi5zzc-MuiCircularProgress-root"
      data-testid="circular-progress"
      role="progressbar"
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </span>
  </div>
</div>
`;

exports[`renders App Component 1`] = `
<div
  className="App"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg css-14uxf7f-MuiContainer-root"
  >
    <span
      className="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-1gi5zzc-MuiCircularProgress-root"
      data-testid="circular-progress"
      role="progressbar"
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </span>
  </div>
</div>
`;

exports[`renders the App Component 1`] = `
<div
  className="App"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg css-14uxf7f-MuiContainer-root"
  >
    <span
      className="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-1gi5zzc-MuiCircularProgress-root"
      data-testid="circular-progress"
      role="progressbar"
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </span>
  </div>
</div>
`;
